Floofbot/Modules/Utilities.cs

```diff
@@ -139,5 +141,60 @@ public async Task ServerInfo()
             await Context.Channel.SendMessageAsync("", false, embed.Build());
 
         }
+        [Command("serverlist")]
+        [Summary("Returns a list of servers that the bot is in")]
+        [RequireOwner]
+        public async Task ServerList()
+        {
+            List<SocketGuild> guilds = new List<SocketGuild>(Context.Client.Guilds);
+            List<PaginatedMessage.Page> pages = new List<PaginatedMessage.Page>();
+
+            foreach (SocketGuild g in guilds)
+            {
+                List<EmbedFieldBuilder> fields = new List<EmbedFieldBuilder>();
+
+                fields.Add(new EmbedFieldBuilder()
+                {
+                    Name = $"Owner",
+                    Value = $"{g.Owner.Username}#{g.Owner.Discriminator} | ``{g.Owner.Id}``",
+                    IsInline = false
+                });
+                fields.Add(new EmbedFieldBuilder()
+                {
+                    Name = $"Server ID",
+                    Value = g.Id,
+                    IsInline = false
+                });
+                fields.Add(new EmbedFieldBuilder()
+                {
+                    Name = $"Members",
+                    Value = g.MemberCount,
+                    IsInline = false
+                });
+
+                pages.Add(new PaginatedMessage.Page
+                {
+                    Author = new EmbedAuthorBuilder { Name = g.Name },
```

Comment:
Actually, what does Author here do?
Is it like the "Title" field?

Reply:
Kinda like the title per page